std/concurrency.d

```diff
@@ -1716,15 +1716,7 @@ void yield(T)(T value)
 }
 
 
-version (Win64)
-{
-    // fibers are broken on Win64
-}
-else version (Win32)
-{
-    // fibers are broken in Win32 under server 2012: bug 13821
-}
-else unittest
+unittest
 {
```

Comment:
the PR got merged and this PR passes the autotester. So I would conclude it is fixed.